proxy/src/main/java/com/wavefront/agent/handlers/DeltaCounterAccumulationHandlerImpl.java

```diff
@@ -84,7 +92,9 @@ public DeltaCounterAccumulationHandlerImpl(
         aggregationIntervalSeconds, TimeUnit.SECONDS);
 
     String metricPrefix = handlerKey.toString();
-    this.reportedCounter = Metrics.newCounter(new MetricName(metricPrefix, "", "sent"));
+    MetricName reported = new MetricName(metricPrefix, "", "sent");
+    this.reportedCounter = Metrics.newCounter(reported);
+    this.reportedStats = new BurstRateTrackingCounter(reported, Metrics.defaultRegistry(), 1000);
```

Comment:
what's actually updating this.reportStats?
when reportedCounter is incremented, does reportedStats also update, due to the shared MetricName ?

Reply:
shouldn't the increment in reportAggregatedDeltaValue be changed to this.reportStats.inc()?